scripts/meta/gatherer.py

```diff
@@ -0,0 +1,42 @@
+import json
+from website.settings import GITHUB_API_TOKEN
+
+the_filename = 'website/static/git_logs.json'
+
+
+def gather_pr_data():
+    import requests
```

Comment:
Travis disagrees with you on this. https://travis-ci.org/CenterForOpenScience/osf.io/builds/122245666